docker-compose.yml

```diff
@@ -3,7 +3,10 @@ services:
 
   artifactory:
     image: docker.bintray.io/jfrog/artifactory-pro:6.2.0
+    environment:
+      - EXTRA_JAVA_OPTIONS="-agentlib:jdwp=transport=dt_socket,address=8082,server=y,suspend=n"
```

Comment:
Is is for debug? Should it be here?

Reply:
@Kirill89, strictly speaking, no. This is enabling debug mode (so I can connect from IDE) for dev artifactory instance. Do we want to exclude it to separate PR?

Reply:
It's okay to have it in this PR 🙏